tests/test_utils.py

```diff
@@ -348,6 +349,13 @@ def first_class(true_labels, model_labels, idx=0):
         self.assertEqual(performance_diff(full_model, limited_model, x_test, y_test, perf_function=first_class, idx=1),
                          -1.0 / 3)
 
+    def test_is_probability(self):
+        x = np.array([0.1, 0.3, 0.5999])
+        y = np.array([0.1, 0.3, 0.599])
+
```

Comment:
Please add `assertFalse` tests for negative values and for values larger than one

Reply:
Solved.